pulp_ansible/app/migrations/0029_manifest_and_files_json_fields.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 2.2.17 on 2020-12-10 15:04
+import json
+import tarfile
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations
+
+
+def _get_file_obj(tar, file_path, artifact_path):
```

Comment:
With this being present in a few places I think the real implementation should be moved to a utility area.